test/core/integration/Integration.t.sol

```diff
@@ -257,4 +268,133 @@ abstract contract Integration_Test is Base_Test {
+        returns (uint256 streamId)
+    {
+        streamId = lockup.createWithDurationsLD{ value: 0 }(params, segmentsWithDuration);
+    }
+
+    function createWithDurationsLL(
+        Lockup.CreateWithDurations memory params,
+        LockupLinear.UnlockAmounts memory unlockAmounts,
+        LockupLinear.Durations memory durations
+    )
+        internal
+        returns (uint256 streamId)
+    {
+        streamId = lockup.createWithDurationsLL{ value: 0 }(params, unlockAmounts, durations);
+    }
+
+    function createWithDurationsLT(
+        Lockup.CreateWithDurations memory params,
+        LockupTranched.TrancheWithDuration[] memory tranchesWithDuration
+    )
+        internal
+        returns (uint256 streamId)
+    {
+        streamId = lockup.createWithDurationsLT{ value: 0 }(params, tranchesWithDuration);
+    }
+
+    function createWithTimestampsLD(
+        Lockup.CreateWithTimestamps memory params,
+        LockupDynamic.Segment[] memory segments
+    )
+        internal
+        returns (uint256 streamId)
+    {
+        streamId = lockup.createWithTimestampsLD{ value: 0 }(params, segments);
+    }
+
+    function createWithTimestampsLL(
+        Lockup.CreateWithTimestamps memory params,
+        LockupLinear.UnlockAmounts memory unlockAmounts,
+        uint40 cliffTime
+    )
+        internal
+        returns (uint256 streamId)
+    {
+        streamId = lockup.createWithTimestampsLL{ value: 0 }(params, unlockAmounts, cliffTime);
+    }
+
+    function createWithTimestampsLT(
+        Lockup.CreateWithTimestamps memory params,
+        LockupTranched.Tranche[] memory tranches
+    )
+        internal
+        returns (uint256 streamId)
+    {
+        streamId = lockup.createWithTimestampsLT{ value: 0 }(params, tranches);
+    }
+
+    function renounce(uint256 streamId) internal {
+        lockup.renounce{ value: 0 }(streamId);
+    }
+
+    modifier balanceTest() {
+        uint256 balanceBefore = address(lockup).balance;
+        _;
+        assertEq(address(lockup).balance, balanceBefore + SABLIER_FEE, "balance after function call");
+    }
+
+    function withdraw(uint256 streamId, address to, uint128 amount) internal {
+        lockup.withdraw{ value: SABLIER_FEE }(streamId, to, amount);
+    }
+
+    function withdrawWithBalTest(uint256 streamId, address to, uint128 amount) internal balanceTest {
+        withdraw(streamId, to, amount);
+    }
+
+    function withdrawMax(uint256 streamId, address to) internal returns (uint128) {
+        return lockup.withdrawMax{ value: SABLIER_FEE }(streamId, to);
+    }
+
+    function withdrawMaxWithBalTest(uint256 streamId, address to) internal balanceTest returns (uint128) {
+        uint128 withdrawn = withdrawMax(streamId, to);
+        return withdrawn;
+    }
+
+    function withdrawMaxAndTransfer(uint256 streamId, address newRecipient) internal returns (uint128) {
+        return lockup.withdrawMaxAndTransfer{ value: SABLIER_FEE }(streamId, newRecipient);
```

Comment:
I'd be in favour of removing these helpers and use the function itself where it is needed with a non-zero `value`

Reply:
Fair enough, agree